### PR TITLE
[MU3] Fix MSVC compiler warnings

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**

### CODE REVIEW

File: libmscore/measure.cpp [hunk 1]
```diff
@@ -2897,7 +2897,7 @@ bool Measure::isCutawayClef(int staffIdx) const
             etrack = strack + VOICES;
             }
       // find segment before EndBarLine
-      Segment* s;
+      Segment* s = nullptr;
```
Comment (Jojo-Schmitz) [inline libmscore/measure.cpp:2900]: same as in #7241

File: mscore/musescore.h [hunk 2]
```diff
@@ -684,7 +684,7 @@ class MuseScore : public QMainWindow, public MuseScoreCore {
       void loadFile(const QUrl&);
       QTemporaryFile* getTemporaryScoreFileCopy(const QFileInfo& info, const QString& baseNameTemplate);
       QNetworkAccessManager* networkManager();
-      virtual Score* openScore(const QString& fn, bool switchTab = true, const bool appendToExistingTabs = true);
+      virtual Score* openScore(const QString& fn, bool switchTab = true, const bool appendToExistingTabs = true) override;
```
Comment (Jojo-Schmitz) [inline mscore/musescore.h:687]: Review needed, it does fix a compiler warning but I'm not sure whether we really want to override `Ms::MuseScoreCore::openScore()` from libmscore/musescoreCore.h line 45 here?

Comment (Howard-C) [reply mscore/musescore.h:687]: Does it matter? I think it is going to be overridden with or without the "override" keyword

Comment (Jojo-Schmitz) [reply mscore/musescore.h:687]: It was implicitly overridden, is now explcitly overridden, that apeases the compiler. Question is whether this really wanted.

Comment (Howard-C) [reply mscore/musescore.h:687]: I'm just saying even if that wasn't what we want, it still would not be solved (returned to the behaviour of parent class) just by not adding `override`. Am I wrong?

Comment (Jojo-Schmitz) [reply mscore/musescore.h:687]: No. And that's why I started this discussion here ;-)

**Pull request merged**